examples/node-pub

```diff
@@ -27,19 +28,19 @@ const subject = argv._[0]
 const msg = argv._[1] || ''
 
 if (!subject) {
-  console.log('Usage: node-pub  [-s server] [--creds file] <subject> [msg]')
+  console.log('Usage: node-pub  [-s server] [--creds=filepath] <subject> [msg]')
```

Comment:
X=Y syntax required?

Reply:
yes for flags that are not a single letter